src/dataflow/sink/kafka.rs

```diff
@@ -85,13 +85,26 @@ pub fn kafka<G>(
             let producer: FutureProducer = config.create().unwrap();
 
             stream.sink(Pipeline, &format!("kafka-{}", id), move |input| {
-                let encoder = Encoder::new(&schema.to_string());
                 input.for_each(|_, rows| {
                     for (row, _time, diff) in rows.iter() {
-                        let buf = encoder.encode(schema_id, row, *diff);
-                        let record: FutureRecord<&Vec<u8>, _> =
-                            FutureRecord::to(&connector.topic).payload(&buf);
-                        producer.send(record, 1000 /* block_ms */);
+                        let diff_pair = if *diff < 0 {
```

Comment:
I think we might be better off logging an error than emitting an insert if diff = 0?

Reply:
yup yup I saw that just thought emitting some kind of error might be more debug friendly than silently doing nothing